README.md

```diff
@@ -55,19 +55,36 @@ This library provides the collection of utils classes.
 
```

Comment:
the xml snippet added is not relevant to the library. Please remove

Reply:
done